moveit_core/collision_detection_bullet/src/collision_env_bullet.cpp

```diff
@@ -89,6 +89,9 @@ CollisionEnvBullet::CollisionEnvBullet(const CollisionEnvBullet& other, const Wo
   {
```

Comment:
If you actually use this constructor and it works for you with this PR, please remove the `TODO` to verify it above.

Reply:
Removed in 5495a9c